src/tests/ftest/pool/evict_test.py

```diff
@@ -260,15 +273,16 @@ def test_evict(self):
                             count+1, pool[count].uuid, c_uuid_to_str(
                                 pool_info.pi_uuid)))
 
-    def test_evict_bad_server_name(self):
-        """
-        Test evicting a pool using an invalid server group name.
+    # Commented out due to DAOS-5545
+    #def test_evict_bad_server_name(self):
+    #    """
+    #    Test evicting a pool using an invalid server group name.
 
-        :avocado: tags=all,pool,pr,full_regression,small,poolevict
-        :avocado: tags=poolevict_bad_server_name,DAOS_5610
-        """
-        test_param = self.params.get("server_name", '/run/badparams/*')
-        self.assertTrue(self.evict_badparam(test_param))
+    #    :avocado: tags=all,pool,pr,full_regression,small,poolevict
+    #    :avocado: tags=poolevict_bad_server_name,DAOS_5610
+    #    """
+    #    test_param = self.params.get("server_name", '/run/badparams/*')
+    #    self.assertTrue(self.evict_badparam(test_param))
```

Comment:
If it can be avoided, we should not be commenting out tests, but rather skipping them with `skipForTicket()` or `cancelForTicket()`.  When we comment out tests we lose the ability to track that they are not being run.  Since in this case we want to skip running an entire test method, the `skipForTicket()` should be used:
```
    @skipForTicket("DAOS-5545")
    def test_evict_bad_server_name(self):
```

Reply:
Thank you.